### ISSUE TITLE
Feature request: warn users of missing image description

### ISSUE THREAD

**Issue by IsabelaY** [mastodon/mastodon#7804]
The issue: it's not immediately obvious that you can add an image description to the image, so I've seen a lot of people forgetting to add image description, then having to delete/redraft, which can delete existing replies/retoots unnecessarily.

Possible solution: warn the users that there are images that are missing description. It does not need to block the "Toot!" button.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by purpliminal** [comment]
How about making the UI to add the image description more prominent? I feel like making it always show up, instead of just on hover, would make it less likely to forget.

**Comment by QueerNeko** [comment]
Yeah, I think that would be enough.

**Comment by BenLubar** [comment]
![the three "show confirmation before" settings](https://user-images.githubusercontent.com/4257305/47956693-8678d180-df76-11e8-8806-6ac015c25cf6.png)

Seems like it would fit well into this section.

**Comment by thejamesm** [comment]
Yeah, I'd like an optional prompt when there's no description. Some would find that intrusive, but I'm forgetful and would appreciate the reminder, so it seems like an optional dialogue is the way to go.

**Comment by paniq** [comment]
any news on this one?

**Comment by Igigog** [comment]
+1

**Comment by ccazabon** [comment]
While making the add-alt-text UI element show up all the time is great, I still find myself submitting a post without adding descriptive text to the media.

It would really help me, and perhaps others, to have an (opt-in?) confirmation dialog for posting media without descriptive text.

:arrow_up:
